deploy-manage/monitor/autoops/ec-autoops-dismiss-event.md

```diff
@@ -10,9 +10,4 @@ products:
 
 # Dismiss Events [ec-autoops-dismiss-event]
 
-Some events that are triggered may not require your attention immediately, or at all. Users with the appropriate permissions can choose to dismiss an event, preventing AutoOps from opening it. This action can be reversed using the Dismiss events report.
-
-:::{image} /deploy-manage/images/cloud-autoops-dismiss-events.png
-:alt: Dismiss events
-:::
-
+This page has moved. Refer to [Event actions](/deploy-manage/monitor/autoops/ec-autoops-events.md#event-actions) to learn about what you can do with events, including how to dismiss them.
```

Comment:
Is this a placeholder for a follow up PR? What's the benefit of this instead of using a redirect?